domain/src/main/java/org/oppia/android/domain/platformparameter/PlatformParameterSingleton.kt

```diff
@@ -0,0 +1,53 @@
+package org.oppia.android.domain.platformparameter
```

Comment:
I am ignoring this file because it was reviewed in another PR. If theres anything I should see here please let me know!

Reply:
Yes you can ignore. Also i have listed the files which needs to be reviwed for this PR (ie the files that are not copied from previous one and are associated with this PR only)